dbt/adapters/bigquery/connections.py

```diff
@@ -437,51 +439,67 @@ def execute(
         code = None
         num_rows = None
         bytes_processed = None
+        job_id = None
+        project_id = None
+        num_rows_formatted = None
+        processed_bytes = None
+
+        def set_common_attributes():
+            nonlocal message, num_rows, num_rows_formatted, bytes_processed, processed_bytes, job_id, project_id
+            bytes_processed = query_job.total_bytes_processed
+            processed_bytes = self.format_bytes(bytes_processed)
+            job_id = query_job.job_id
```

Comment:
At least for the tabbed BigQuery Console UI in in EU, I need a qualified job id.
```suggestion
            job_id = f"bq:{query_job.location}:{query_job.job_id}"
```

Reply:
Thanks for the feedback, as I'm running in US region, I didn't know about it.
Yet I would put the location in a field and add the prefixes in the logged url as some people might use the returned value in adapter response.

Reply:
I've added the location in the response & log 👍